### DISCUSSION TITLE
PortalGenerationRequiresPermission for 1.14+

### DISCUSSION THREAD

**Discussion by NullCase** [GriefPrevention/GriefPrevention#546] [Ideas]
It seems like you might already have an idea for this, but here's a post. Worst case it can be a reminder.

Based on https://github.com/TechFortress/GriefPrevention/issues/534 it looks like Spigot removed the event that GP used to perform this function in 1.13.2.

I don't know. Do you think GP will lose this feature after 1.14?

----------

Test: 
- generate config for GP 16.11.6 and set PortalGenerationRequiresPermission: true. 
- Generate portal that maps to claimed land where you don't have permission. 
- Portal will not generate on the claimed land, respecting permissions.

Upgrade to GP 16.12.alpha
- Note that PortalGenerationRequiresPermission is scrapped in the config.
- Generate portal that maps to claimed land.
- Portal generates on claimed land, as expected.

Request: some way to add PortalGenerationRequiresPermission again.

**Comment by RoboMWM** [comment]
Can anyone confirm if PlayerPortalEvent has taken on the methods that used to exist in the TravelAgent? Because it looks like that's what happened, I think, after briefly checking the javadocs?
